lib/src/api/opt/auth.rs

```diff
@@ -12,6 +13,108 @@ pub struct Signup;
 #[derive(Debug)]
 pub struct Signin;
 
+/// Credentials level
+#[derive(Debug, Clone)]
+pub enum CredentialsLevel {
```

Comment:
Does this need to be part of the API? Are exposing it anywhere in the SDK? If not, can we hide it too?

Reply:
Thank you! I have moved these types to the CLI in [526a849](https://github.com/surrealdb/surrealdb/pull/2985/commits/526a849fa33e5194ac207ffc69d2ead389e98c28).